src/Plugin/EncryptionMethod/McryptAES128Encryption.php

```diff
@@ -14,23 +14,11 @@
  * @EncryptionMethod(
  *   id = "mcrypt_aes_128",
  *   title = @Translation("Mcrypt AES 128"),
- *   description = "This uses PHPs mcrypt extension and <a href='http://en.wikipedia.org/wiki/Advanced_Encryption_Standard'>AES-128</a>."
+ *   description = "This uses PHPs mcrypt extension and <a href='http://en.wikipedia.org/wiki/Advanced_Encryption_Standard'>AES-128</a>.",
+ *   key_type = {"aes_encryption"}
  * )
  */
-class McryptAES128Encryption extends PluginBase implements EncryptionMethodInterface {
-
-  /**
-   * @return mixed
-   */
-  public function getDependencies() {
-    $errors = array();
-
-    if (!function_exists('mcrypt_encrypt')) {
-      $errors[] = t('MCrypt library not installed.');
-    }
-
-    return $errors;
-  }
+class McryptAES128Encryption extends EncryptionMethodBase implements EncryptionMethodInterface {
```

Comment:
This should really be its own contrib module and not just in TFA - other folks can use this